java/vector/src/main/java/org/apache/arrow/vector/BaseVariableWidthVector.java

```diff
@@ -1176,14 +1176,25 @@ protected final void setBytes(int index, byte[] value, int start, int length) {
     /* end offset of current last element in the vector. this will
      * be the start offset of new element we are trying to store.
      */
-    final int startOffset = getstartOffset(index);
+    final int startOffset = getStartOffset(index);
     /* set new end offset */
     offsetBuffer.setInt((index + 1) * OFFSET_WIDTH, startOffset + length);
     /* store the var length data in value buffer */
     valueBuffer.setBytes(startOffset, value, start, length);
   }
 
+  /**
+   * Gets the starting offset of a record, given its index.
+   * This method is deprecated. Please use {@link BaseVariableWidthVector#getStartEnd(int)} instead.
```

Comment:
minor nit! getStartEnd() -> getStartOffset()..

Reply:
Revised. Thanks for the careful review.